### PR TITLE
Add vignette for grading with multiple solutions

### LINKED ISSUE #292: Add more documentation for multiple solutions (#289)
Probably best to hold off until #296 is finished.

### PR DESCRIPTION
**Pull request by rossellhayes**
Potentially closes #292.

### CODE REVIEW

File: vignettes/multiple_solutions.Rmd [hunk 1]
```diff
@@ -0,0 +1,462 @@
...
+```{css echo=FALSE}
+.alert :last-child {
+  margin-bottom: 0;
+}
+```
+
+## Overview
+
+Sometimes we want to provide feedback for exercises that have multiple possible solutions.
+With a little setup, gradethis can handle exercises with multiple solutions just as well as any other!
```
Comment (gadenbuie) [inline vignettes/multiple_solutions.Rmd:75]:
```suggestion
Not all exercises have exactly one right answer:
there could be a few different ways to arrive at the same result,
or the exercise prompt could be open-ended and ask the student to pick from a selection of possibilities.
In either case, and with minimal changes to your learnr tutorial,
gradethis can accommodate exercises with multiple solutions!
```

**Review by gadenbuie — COMMENTED** [review]
This is a great start, thanks! I was going to to try to provide edits as suggestions, but it's too hard to do through the UI, so I'm going to work locally and commit edits to the PR branch

File: vignettes/multiple_solutions.Rmd [hunk 2]
```diff
@@ -0,0 +1,523 @@
...
+  fail_if_code_feedback()
+  pass_if_equal()
+  fail(hint = TRUE)
+})
+
+grade_round_example <- grade_example_factory(solution_code, grader)
+```
+
+If a student enters any of the correct answers to arrive at the value `5`,
+they recieve positive feedback and encouragement.
```
Comment (rossellhayes) [inline vignettes/multiple_solutions.Rmd:216]:
```suggestion
they receive positive feedback and encouragement.
```

File: vignettes/multiple_solutions.Rmd [hunk 3]
```diff
@@ -0,0 +1,523 @@
...
+```
+# code header ----
+```
+
+You can think of the code headers as _labels_ for each solution variant.
+(The [full exercise](#example-different-results) is presented below.)
+
+````markdown
+```{r average-solution}`r ''`
+`r code_chunk("average-solution")`
```
Comment (rossellhayes) [inline vignettes/multiple_solutions.Rmd:110]:
```suggestion
`r chunk_code("average-solution")`
```

**Review by rossellhayes — COMMENTED** [review]
Two quick copy edits

File: vignettes/multiple_solutions.Rmd (hunk 3 above)
Comment (gadenbuie) [reply vignettes/multiple_solutions.Rmd:110]: 🤦

**Review by gadenbuie — APPROVED** [review]
👨‍🎤 🎸

**Pull request merged**